non-oss/pharos_pro/addons/kontena-lens/addon.rb

```diff
@@ -36,14 +36,15 @@
       email: config.tls&.email,
       user_management: user_management_enabled?
     )
-    wait_for_dashboard(host)
+    protocol = config.host && config.tls&.email ? 'http' : 'https' # with cert-manager we have to use http since tls secret is not in place immediately
```

Comment:
Wat? Really hard to read this line... does this mean that `protocol` is `nil` if host is not set?
```suggestion
    protocol = config.tls&.email ? 'http' : 'https' if config.host # with cert-manager we have to use http since tls secret is not in place immediately
```

Reply:
if `config.host` and `config.tls.email` is defined then protocol is`http` , otherwise `https`. 
With your suggestion protocol is set to nil unless `config.host` is defined, which is wrong.

Now thinking that actually we need only check if `config.tls.email` is set, then use `http`, otherwise `https`.

Reply:
```suggestion
    protocol = (config.host && config.tls&.email) ? 'http' : 'https' # with cert-manager we have to use http since tls secret is not in place immediately
```

Reply:
rubocop is not happy with that

Reply:
Ok, rubocop is the law but it looks so wrong to my eyes... maybe too little ☕️ .

Reply:
I will change that to check only `config.tls&.email`. I will also tweak ingress and certificate resources a bit. With these tweaks user is able to fetch valid cert for `lens.<worker-node-ip>.nip.io` too, which is nice.